lib/main.js

```diff
@@ -57,20 +86,22 @@ export default {
   addOpener() {
     this.disposables.add(atom.workspace.addOpener((uri) => {
       if (uri === TERMINAL_TAB_URI) {
+        this.lastActivePane = atom.workspace.getActivePane();
         return new TerminalSession();
       }
-    }));
+    }.bind(this)));
```

Comment:
I'd prefer not to have this bind, I couldn't find an elegant way to store history information.

I looked around for an API for Atom to return back which panes were previously active, but I couldn't find one. If there was, or a function which just moves focus back to a previous pane, I can remove any state stored in the plugin itself.